seed_data/challenges/computer-science.json

```diff
@@ -67,8 +76,12 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende sobre el Hardware de las Computadoras",
+      "descriptionEs": [
+        "La semana 3 del curso de Introducción a Informática de Stanford cubrira el hardware de las computadoras y explicará sobre la ley de Moore y el crecimiento exponencial del precio-desempeño de los procesadores.",
```

Comment:
Little enhancements to this translation:
1. "semana 3" should be "Semana 3" to keep consistency with the previous waypoint.
2. "cubrira" should be "cubrirá" (with accented á at the end)
3. The Moore's law is about the exponential growth, so the phrase about that should be "y explicará la ley de Moore sobre el crecimiento exponencial del precio-desempeño de los procesadores".